File: sample/src/main/java/me/xiaopan/assemblyadaptersample/fragment/ImageFragment.java
package me.xiaopan.assemblyadaptersample.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import me.xiaopan.assemblyadaptersample.R;
import me.xiaopan.sketch.SketchImageView;

public class ImageFragment extends Fragment {

    private String imageUrl;

    public static Bundle buildParams(String imageUrl) {
        Bundle bundle = new Bundle();
        bundle.putString("imageUrl", imageUrl);
        return bundle;
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Bundle params = getArguments();
        if (params != null) {
            imageUrl = params.getString("imageUrl");
        }
    }

    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_image, container, false);
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        SketchImageView imageView = (SketchImageView) view.findViewById(R.id.image_imageFragment);
        imageView.displayImage(imageUrl);
    }
}

File: sample/src/main/java/me/panpf/adapter/sample/fragment/ImageFragment.kt
package me.panpf.adapter.sample.fragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import me.panpf.sketch.SketchImageView
import me.panpf.adapter.sample.R
import me.panpf.adapter.sample.bindView

class ImageFragment : Fragment() {

    private var imageUrl: String? = null
    private val imageView: SketchImageView by bindView(R.id.image_imageFragment)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val params = arguments
        if (params != null) {
            imageUrl = params.getString("imageUrl")
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_image, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        imageView.displayImage(imageUrl ?: "")
    }

    companion object {

        fun buildParams(imageUrl: String): Bundle {
            val bundle = Bundle()
            bundle.putString("imageUrl", imageUrl)
            return bundle
        }
    }
}
